runconfig/opts/parse.go

```diff
@@ -13,11 +13,11 @@ import (
 	flag "github.com/docker/docker/pkg/mflag"
 	"github.com/docker/docker/pkg/mount"
 	"github.com/docker/docker/pkg/signal"
+	"github.com/docker/docker/vendor/src/github.com/docker/go-units"
```

Comment:
I've been having trouble with my vim go-plugins setup making this same change; only on `go-units`  Haven't figured out what is special, although one time I got frustrated enough to play with GOPATH and other bits until I got it to not change it for that session :(